docs/UpdateButton.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: default
+title: "The UpdateButton Component"
+---
+
+# `<UpdateButton>`
+
+This component allows to create a button that updates a record by calling the [`useUpdate hook`](./useUpdate.md).
+
+<video controls playsinline muted loop poster="./img/updatebutton.png" >
+  <source src="./img/updatebutton.webm" type="video/mp4" />
```

Comment:
You're mixing up webm and mp4 on this line. Btw you should have both or only mp4.